src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx

```diff
@@ -442,8 +442,8 @@ export class FileBrowserDialogComponent extends React.Component {
                 minimal={true}
                 content={
                     <Menu>
-                        <MenuItem text="CRTF Region File" onClick={() => fileBrowserStore.setExportFileType(CARTA.FileType.CRTF)} />
-                        <MenuItem text="DS9 Region File" onClick={() => fileBrowserStore.setExportFileType(CARTA.FileType.DS9_REG)} />
+                        <MenuItem text="CRTF region file" onClick={() => fileBrowserStore.setExportFileType(CARTA.FileType.CRTF)} />
+                        <MenuItem text="DS9 region File" onClick={() => fileBrowserStore.setExportFileType(CARTA.FileType.DS9_REG)} />
```

Comment:
sorry I should explain more detail, because "CRTF region" using "file", however "DS9 region" using "File", should these two file being consistant?

Reply:
also fixed